run/run_experiments.sh

```diff
@@ -270,9 +270,12 @@ push_and_run() {
 
     # Build CPISync on remote
     ssh $address "cd $path/$(basename $cpisync_path)
-                  mkdir build && cd build
```

Comment:
What if the build directory already exists?  Why remove the check?

Reply:
I'm confused... That's what these changes are doing.

Reply:
If the directory already exists, mkdir will give an error, which may be undesired.  Consider using -p